manpages.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var manpagesMatcher = regexp.MustCompile(`\b(\w+)\((\d+)\)(\W|$)`)
+
+func runnerManpages(parsed Message) {
+	for _, l := range extractManpages(parsed.Trailing) {
+		Privmsg(Target(parsed), "[manpage] "+l)
+	}
+}
+
+func extractManpages(msg string) (links []string) {
+	msg = strings.TrimPrefix(msg, "frank: ")
```

Comment:
I can easily see people writing e.g. “man cherti”. Maybe we should require the “frank: ” prefix?